dora/src/masm/x64.rs

```diff
@@ -1295,6 +1295,62 @@ impl MacroAssembler {
         }
     }
 
+    pub fn float_round_tozero(&mut self, mode: MachineMode, dest: FReg, src: FReg) {
+        if !has_round() {
+            panic!("unsupported operation")
```

Comment:
What should we do when those instructions aren't supported? Do we need a workaround? From what I could find online in a minute or so, it seems we can safely assume those instructions exist since they seem old enough. In this case I would just let the CPU trap on these instructions and get rid of `has_round()` completely. Wdyt?

Reply:
My intention was to leave the door open for an asm fallback implementation, but I think that requiring a CPU from 2007 or later is reasonable. I'll drop that code.